src/search/services/search.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { Board } from 'src/boards/entities/board.entity';
+import { User } from 'src/users/entities/user.entity';
+import { EntityManager } from 'typeorm';
+
+@Injectable()
+export class SearchService {
+  constructor(private entityManager: EntityManager) {}
+  async searchBoardsAsHeadOrBody(searchQuery: string) {
+    const boardRepository = this.entityManager.getRepository(Board);
+
+    return boardRepository
+      .createQueryBuilder()
+      .select()
+      .where(`MATCH(head) AGAINST (:searchQuery)`, {
+        searchQuery: `${searchQuery}`,
```

Comment:
```suggestion
        searchQuery,
```
이렇게해도될거같은데 백틱으로 감싼 이유가 있나요??

Reply:
이전에 IN BOOLEAN MODE 사용할 때 * 연산자를 붙이려고 해놨었는데 지우는걸 깜빡 했습니다!